actionlist/app/containers/ActionListContainer.js

```diff
@@ -1,17 +1,69 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import {
   View,
+  DrawerLayoutAndroid,
 } from 'react-native';
 
 import Header from '../components/header';
 import DisplayList from '../components/display_list';
+import SideMenu from '../components/SideMenu';
+import { toggleDrawer } from '../actions/action_items';
+
+class ActionListContainer extends Component {
```

Comment:
Right. So you wrap `DrawerLayoutAndroid` in a stateful component and then render that stateful component in ActionListContainer (and pass `drawerExpanded` as a property and maybe `SideMenu` as a child or as a property, since `DrawerLayoutAndroid` expects a no-arg function for whatever reason). Basically you're just extracting the `connect` call (and the related map to props functions) to another component so you don't mix Redux and stateful components.